recipe/meta.yaml

```diff
@@ -7,42 +7,39 @@ package:
 source:
   fn: libcmaes-{{ version }}.tar.gz
   url: https://github.com/beniz/libcmaes/archive/{{ version }}.tar.gz
-  sha256: 899fcc44b551e4976a77e48e41bdd5dafc3394bd3fcb3c55996c02e4267bfa28
-  patches:
-    # https://github.com/beniz/libcmaes/pull/149
-    - py3k.patch
+  sha256: 1b23e53eb4acb123348e7d00dfdd7c7699c513950347fcde7425376ad333f6c5
 
 build:
-  number: 3
+  number: 0
   # does not build with vc9
-  skip: true  # [win and py<35]
+  skip: true  # [win or py<35]
```

Comment:
What is the issue with Windows?

Reply:
I have no idea. The osx build is working fine now after patching in the dynamic lookup as proposed by @jschueller. Does the MSVC linker also need a similar dynamic lookup flag? I have no idea why the Windows tests are failing.

Reply:
no, see some tests are succeeding, so the module should import ok
so it's something else

Reply:
OK, fixed. All platforms are passing.
@xhochy can you give it another look?